Contact-us/contact-us.html

```diff
@@ -0,0 +1,143 @@
+                    </li>
+                    <li class="nav-item active" style="margin-left: 50px;">
+                        <a class="nav-link" href="Contact-us/contact-us.html">Contact us</a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+    </div>
+
+    <!--Showrooms-->
+    <div class="container">
+        <div class="row pt-md-3 pt-3">
+            <div class="showroom col-md-6">
+                <h6 style="vertical-align: middle;">SHOWROOM NGUYEN THI MINH KHAI</h6>
+                <p>
+                    Address: 264A - 264B - 264C Nguyen Thi Minh Khai, Ward 6, District 3, Ho Chi Minh City, Vietnam. Hotline: 18006867 - 02873016867 Open time: 08h00 - 21h30
+                </p>
+            </div>
+            <div class="mapouter col-md-6">
+                <div class="gmap_canvas"><iframe width="100%" height="300px" id="gmap_canvas" src="https://maps.google.com/maps?ll=10.773922,106.6894049&q=264A Nguyễn Thị Minh Khai&t=&z=14&ie=UTF8&iwloc=&output=embed" frameborder="0" scrolling="no" marginheight="0" marginwidth="0"></iframe>
+                    <a href="https://techwithlove.com/"></a>
+                </div>
+            </div>
+        </div>
+
+        <div class="row pt-md-3 pt-3">
+            <div class="showroom col-md-6">
+                <h6 style="vertical-align: middle;">SHOWROOM PHU LOI</h6>
+                <p>
+                    Address: 408 Binh Duong Highway, Ward Phu Loi, Thu Dau Mot City, Vietnam. Hotline: 18006867 - 02873016867 Open time: 08h00 - 21h30
+                </p>
+            </div>
+            <div class="mapouter col-md-6">
+                <div class="gmap_canvas"><iframe width="100%" height="300px" id="gmap_canvas" src="https://maps.google.com/maps?ll=10.9864469,106.6655436&q=408 Đại lộ Bình Dương&t=&z=14&ie=UTF8&iwloc=&output=embed" frameborder="0" scrolling="no" marginheight="0" marginwidth="0"></iframe>
+                    <a href="https://techwithlove.com/"></a>
+                </div>
+            </div>
+        </div>
+
+        <div class="row pt-md-3 pt-3">
+            <div class="showroom col-md-6">
+                <h6 style="vertical-align: middle;">SHOWROOM NGUYEN OANH</h6>
+                <p>
+                    Address: 2A Nguyen Oanh, Ward 7, District Go Vap, Ho Chi Minh City, Vietnam. Hotline: 18006867 - 02873016867 Open time: 08h00 - 21h30
+                </p>
+            </div>
+            <div class="mapouter col-md-6">
+                <div class="gmap_canvas"><iframe width="100%" height="300" id="gmap_canvas" src="https://maps.google.com/maps?ll=10.8304687,106.6782603&q=2a Nguyễn Oanh&t=&z=14&ie=UTF8&iwloc=&output=embed" frameborder="0" scrolling="no" marginheight="0" marginwidth="0"></iframe>
+                    <a href="https://techwithlove.com/"></a>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <!--Footer-->
+    <section id="footer">
+        <div class="container">
```

Comment:
Using one `<div class="container">` for the page

Reply:
done